conf/deploydb.sh

```
@@ -22,20 +22,44 @@ if [[ `id -u` -ne 0 ]] && [[ x"$user" = x"root" ]]; then
 
-mysql_run << EOF
-DROP DATABASE IF EXISTS zstack;
-CREATE DATABASE zstack;
-DROP DATABASE IF EXISTS zstack_rest;
-CREATE DATABASE zstack_rest;
-grant all privileges on zstack.* to root@'%' identified by "$password";
-grant all privileges on zstack_rest.* to root@'%' identified by "$password";
-grant all privileges on zstack.* to root@'localhost' identified by "$password";
-grant all privileges on zstack_rest.* to root@'localhost' identified by "$password";
+if command -v greatdb &> /dev/null; then
+  ${MYSQL} ${loginCmd} << EOF
```

Shellcheck (0.10.0)

[warning] loginCmd is referenced but not assigned.

(SC2154)